pkg/dockerregistry/server/metrics/prometheus.go

```diff
@@ -20,9 +20,10 @@ const (
 var (
 	requestDurationSeconds = prometheus.NewSummaryVec(
 		prometheus.SummaryOpts{
-			Namespace: namespace,
-			Name:      "request_duration_seconds",
-			Help:      "Request latency in seconds for each operation.",
+			Namespace:  namespace,
+			Name:       "request_duration_seconds",
+			Help:       "Request latency in seconds for each operation.",
+			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
```

Comment:
@dmage For my own education - are there docs on what this map does?

Reply:
Found them:

```
        // Objectives defines the quantile rank estimates with their respective
	// absolute error. If Objectives[q] = e, then the value reported for q
	// will be the φ-quantile value for some φ between q-e and q+e.  The
	// default value is an empty map, resulting in a summary without
	// quantiles.
```

So with this configuration we have the 50% +/- 5%, 90% +/- 1%, and 99% +/- 0.1%

Reply:
@dmage Was the 50/90/99 reporting previous behavior, or is this new?

Reply:
@adambkaplan  it's the old default

Reply:
fancy math / statistics :-)